UNITTESTS/features/cellular/framework/AT/at_cellularnetwork/at_cellularnetworktest.cpp

```diff
@@ -1048,7 +1047,7 @@ TEST_F(TestAT_CellularNetwork, test_AT_CellularNetwork_get_operator_params)
     EXPECT_TRUE(NSAPI_ERROR_OK == cn.get_operator_params(format, ops));
     EXPECT_TRUE(format == 0);
     EXPECT_TRUE(strcmp(ops.op_long, "12345") == 0);
-    EXPECT_TRUE(strlen(ops.op_short) == 0);
+    EXPECT_EQ(strlen(ops.op_short), 0);
```

Comment:
This doesnt make sense but in my env it was failing EXPECT_TRUE style but not when using EXPECT_EQ.

Reply:
strlen expects NULL terminated value, so check that code is doing it.
Some compilers auto assing to \0 and some don't so your failing case is actually a bug in the code
Don't fix unit test but the actual code.

Reply:
Added initialisation in constructor for the char arrays of operator_t.